src/ua/lviv/iot/algo/part1/lab2/Stadium.java

```diff
@@ -0,0 +1,44 @@
+package ua.lviv.iot.algo.part1.lab2;
+
+import lombok.Setter;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import java.util.List;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@ToString(callSuper = true)
+public class Stadium extends AbstractStadium{
+    final static int ONE_HUNDRED_PEOPLE_TO_EXPEL = 100;
```

Comment:
why adding variable with its value in name? Maybe name it "amountOfPeopleToExpel" or something similar?